core/admin.py

```diff
@@ -177,3 +177,16 @@ class WeightAdmin(ImportExportMixin, ExportActionMixin, admin.ModelAdmin):
         "weight",
     )
     resource_class = WeightImportExportResource
+
+
+class TaggedItemInline(admin.StackedInline):
+    model = models.Tagged
+
+
+@admin.register(models.Tag)
+class TagAdmin(admin.ModelAdmin):
```

Comment:
Although I don't think we do this anywhere else... it would nice to override the admin form and set the color field `type` to `color`. See e.g. https://stackoverflow.com/a/39864918

Reply:
I did not even know that this is a thing. I will have to read up on it first... sounds like a good idea nonetheless!

Reply:
Done! :) There now is a nice color-picker in the admin UI.